examples/enable/basic_draw.py

```diff
@@ -2,8 +2,10 @@
 This demonstrates the most basic drawing capabilities using Enable.  A new
 component is created and added to a container.
 """
-from enable.example_support import DemoFrame, demo_main
-from enable.api import Component, Container, Window
+from traits.api import HasTraits, Instance
+from traitsui.api import Item, View
+
+from enable.api import Component, ComponentEditor
 
 
 class Box(Component):
```

Comment:
The `Box` class has a `resizable` attribute set to an empty string. I'm assuming that means that the box is not resizable - can you make this explicit and set it the attr to `False`?

Reply:
The `resizable` trait holds the _directions_ that the trait can be resized, expecting (IIRC) `h` for horizontal and `v` for vertical, `''` means neither direction is resizable.

Reply:
ahh. i shouldn't have made comments without looking at what the trait actually does. sorry for the noise. resolving conversation.